bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx71.html

```diff
@@ -0,0 +1,118 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import monitor_button, fxa_cta_link with context %}
+
+{% add_lang_files "firefox/whatsnew_71" %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ _('What’s new with Firefox - More privacy, more protections.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_71') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew71' %}
+{% set _utm_campaign = 'whatsnew71' %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+      <div class="mzp-c-emphasis-box">
+        <img class="c-emphasis-box-logo" src="{{ static('protocol/img/logos/firefox/logo-md.png') }}" alt="">
+        <h2 class="c-emphasis-box-title">
+          {{ _('No account required. But you might want one.') }}
+        </h2>
+        <p>{{ _('The Firefox browser collects so little data about you, we don’t even require your email address. But when you use it to create a Firefox account, we can protect your privacy across more of your online life.') }}</p>
+
+        <p class="cta">{{ fxa_cta_link(entrypoint=_entrypoint, action='signup', button_text=_('Sign In'), button_class='mzp-c-button mzp-t-product mzp-t-small js-fxa-product-button', utm_campaign=_utm_campaign, utm_content='top-cta') }}</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="content-extra">
+    <div class="mzp-l-content">
+
+      <div class="l-columns-one c-picto-list">
+        <div class="c-picto-block t-adjacent">
+          <div class="c-picto-block-image">
+            <img src="{{ static('img/logos/firefox/logo-monitor.svg') }}" alt="">
+          </div>
+          <h3 class="c-picto-block-title">Firefox Monitor</h3>
```

Comment:
Unwrapped string.

```suggestion
          <h3 class="c-picto-block-title">{{ _('Firefox Monitor') }}</h3>
```

Reply:
These are standalone product names, so should reamin in English. See existing examples here: https://www.mozilla.org/de/firefox/welcome/3/

Reply:
Haha, fair enough!